extensions/cornerstone/src/init.js

```diff
@@ -144,8 +145,12 @@ export default function init({ servicesManager, configuration }) {
     ) {
       if (props.configuration) {
         parsedProps.configuration.drawHandlesOnHover = true;
+        parsedProps.configuration.hideHandlesIfMoving = true;
       } else {
-        parsedProps.configuration = { drawHandlesOnHover: true };
+        parsedProps.configuration = {
```

Comment:
Kind of weird that we don't just do `Object.assign`

Reply:
If it doesn't exist, nothing to assign to, you need to declare the object first.